lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -232,7 +239,10 @@ enum ConstantType : byte
     ctStrictNullDisplay = 7,
     ctTrue = 8,
     ctFalse = 9,
-    ctStringTemplateCallsite = 10
+    ctStringTemplateCallsite = 10,
+    ctPropertyString16 = 11,
+    ctInt16 = 12,
+    ctInt8 = 13,
```

Comment:
why not group the three int size constant types?

Reply:
I guess since the byte code is going to be updated already, I might as well?

Reply:
Sigh... I will have to renumber them all